proposals/0000-android-12-issues.md

```diff
@@ -0,0 +1,291 @@
+# Feature name
+
+* Proposal: [SDL-0000](0000-android-12-issues.md)
+* Author: [Robert Henigan](https://github.com/RHenigan)
+* Status: **Awaiting review**
+* Impacted Platforms: [Java Suite]
+
+## Introduction
+
+This proposal is to address the issues that have been introduced as a result of Android 12.
```

Comment:
```suggestion
This proposal is to address the issues that have been introduced as a result of Android 12.
```
```suggestion
With almost every Android version, Google updates the OS to better distribute the device resources among the running apps to enhance the performance and optimize battery life. However, sometimes the new enhancements come with new restrictions on how apps can use the resources on the device. Android 12 is one of the major Android updates that heavily modify how apps can run in the background and use device resources. This affects how the RouterService starts and communicates with other SDL apps. This proposal is to address the issues that have been introduced as a result of Android 12.
```

Reply:
Should we also mention that it affects how individual apps SdlServices are started?

Reply:
sure. I updated my suggested change.